modules/cluster-logging-collector-log-forwarding-supported-plugins.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forwarding-supported-plugins_{context}"]
+
+= Supported plugins for forwarding logs to third-party systems
+
+[role="_abstract"]
+Red Hat OpenShift Logging version 5.0 tests and supports the following plugins for forwarding logs to third-party systems.
+
+.Plugins
+[options="header"]
+|====
+|Output type|Plugin name|Plugin version|Tested with
+
+| Elasticsearch | link:https://github.com/uken/fluent-plugin-elasticsearch[fluent-plugin-elasticsearch] | link:https://github.com/uken/fluent-plugin-elasticsearch/releases/tag/v4.1.1[4.1.1] | link:https://github.com/openshift/cluster-logging-operator/blob/release-5.0/test/helpers/elasticsearch.go#L219[Elasticsearch 6.8.1]
+
+| Fluent forward protocol | link:https://github.com/fluent/fluentd[fluentd] | link:https://github.com/fluent/fluentd/releases/tag/v1.7.4[1.7.4] | link:https://github.com/openshift/cluster-logging-operator/blob/release-5.0/test/helpers/fluentd/receiver.go#L96[fluent forward v1]
+
+| syslog | link:https://github.com/docebo/fluent-plugin-remote-syslog[fluent-plugin-remote-syslog] | link:https://rubygems.org/gems/fluent-plugin-remote-syslog[1.1] | link:https://github.com/openshift/cluster-logging-operator/blob/release-5.0/test/e2e/logforwarding/syslog/forward_to_syslog_test.go#L156[RFC-5424], link:https://github.com/openshift/cluster-logging-operator/blob/release-5.0/test/e2e/logforwarding/syslog/forward_to_syslog_test.go#L391[RFC-3164]
```

Comment:
the new plugin is https://github.com/dlackty/fluent-plugin-remote_syslog
version is 1.0.0

Reply:
For the "Tested with" column, @vimalk78 is it important to mention what syslog server version we used instead of the RFC standards. Not sure how the RFC standard really influences the different versions. Or maybe we should mention both, the RFC standards we support as well as the syslog server version we have tested this against.

Reply:
Updated. Thanks.